src/bindings/python/CMakeLists.txt

```diff
@@ -41,7 +41,7 @@ function(ov_check_python_build_conditions)
             if(EXISTS ${pybind11_tools_dir})
                 list(APPEND CMAKE_MODULE_PATH ${pybind11_tools_dir})
             else()
-                find_package(pybind11 QUIET)
+                find_package(pybind11 2.8.0 QUIET)
```

Comment:
you can affect many systems with such strict version. Can we extend the range of compatible versions?

Reply:
> you can affect many systems with such strict version

how?

Reply:
discussed offline